main.go

```diff
@@ -17,22 +17,24 @@
 package main
 
 import (
-	log2 "log"
-
-	log "github.com/DataDrake/waterlog"
-	"github.com/DataDrake/waterlog/format"
-	"github.com/DataDrake/waterlog/level"
+	"os"
 
 	_ "github.com/getsolus/solbuild/builder"
 	"github.com/getsolus/solbuild/cli"
+	"github.com/getsolus/solbuild/cli/log"
 )
 
-func init() {
-	log.SetFormat(format.Min)
-	log.SetLevel(level.Info)
-	log.SetFlags(log2.Ltime | log2.Ldate | log2.LUTC)
+func exit() {
+	if r := recover(); r != nil {
+		os.Exit(1)
```

Comment:
Do we need to log what happened here, or is it expected that logging would be covered elsewhere in the program?

Reply:
The latter. This is a temporary measure because of `log.Panic`s (which were fatals) in the code. These should be replaced by returning errors in the code, after which this `exit()` function can be removed.